sites/partners/src/pages/users/index.tsx

```diff
@@ -157,10 +157,10 @@ const Users = () => {
               setSearch: tableOptions.filter.setFilterValue,
             }}
             headerContent={
-              <div className="flex-row">
+              <div className="flex gap-2 items-center">
```

Comment:
In general let's try to add module files if we need to make any tailwind additions so we can do it iteratively, but this is all good for now